nois-testnet-003/genesis.json

```diff
@@ -0,0 +1,329 @@
+{
+  "genesis_time": "2022-10-07T14:07:23.517851339Z",
+  "chain_id": "nois-testnet-003",
+  "initial_height": "1",
+  "consensus_params": {
+    "block": {
+      "max_bytes": "22020096",
+      "max_gas": "4000000",
+      "time_iota_ms": "1000"
```

Comment:
FYI: This seems to be the minimal time difference between two block times:
- https://github.com/tendermint/tendermint/issues/3723
- https://github.com/tendermint/tendermint/pull/3781/files

We need to ensure this is not larger than block times. Otherwise the clock will drift into the future. But with 3s block times and 1s iota we should be fine.